nss-day-cohort-47/tabloidfullstack-wow
# Tl user active

# Description
Add active column to database set default to 1
Users are able to be deactivated and activate

## Type of change


# Testing Instructions
Add active column to database
Active int no nulls default 1
Go to userProfile page. Deactivate a user. Select View Deactivated. A list of deactivated users should display. Select activate user. User should then be reactivated

# Checklist:
